src/libraries/LPFeeLibrary.sol

```diff
@@ -14,6 +14,7 @@ library LPFeeLibrary {
 
     // the lp fee is represented in hundredths of a bip, so the max is 100%
     uint24 public constant MAX_LP_FEE = 1000000;
+    uint24 public constant MAX_UINT24 = type(uint24).max;
```

Comment:
maybe a name like `NO_FEE_CHANGE_FLAG` or something would be clearer? or `NO_FEE_UPDATE_FLAG`? not sure

Reply:
yep i like NO_FEE_UPDATE_FLAG